tasks/kmt.py

```diff
@@ -812,6 +812,50 @@ def kmt_sysprobe_prepare(
     ctx.run(f"ninja -d explain -v -f {nf_path}")
 
 
+def images_matching_ci(ctx, domains):
+    platforms = get_platforms()
+    arch = full_arch("local")
+    kmt_os = get_kmt_os()
+
+    not_matches = list()
+    for tag in platforms[arch]:
+        platinfo = platforms[arch][tag]
+        vmid = f"{platinfo['os_id']}_{platinfo['os_version']}"
+
+        check_tag = False
+        for d in domains:
+            if vmid in d.name:
+                check_tag = True
+                break
+
+        if not check_tag:
+            continue
+
+        manifest_file = '.'.join(platinfo["image"].split('.')[:-2]) + ".manifest"
+
+        if not os.path.exists(f"{kmt_os.rootfs_dir / manifest_file}"):
+            not_matches.append(platinfo["image"])
+            continue
+
+        with open(f"{kmt_os.rootfs_dir / manifest_file}") as f:
+            for line in f:
+                key, value = line.strip().split('=',1)
+                if key != "COMMIT":
```

Comment:
I could add the version to the manifest, that way we would not have to compare with the commit. The main problem with this is that branch images will not match.

Reply:
Hmm we could add a flag to skip this check for when we want to test branch images. Alternatively you could add the version to the manifest directly.

Reply:
ami-builder PR is merged :D Only issue is watching out here for manifests without `IMAGE_VERSION`

Reply:
How would that happen?

Reply:
It wouldn't because you're adding here the manifest files so no one will have outdated manifests. My mistake, sorry :P